lerna.json

```diff
@@ -1,7 +1,14 @@
 {
   "lerna": "3.22.1",
-  "packages": ["packages/*", "samples/*"],
-  "version": "1.3.0-canary.40",
+  "version": "independent",
+  "package": [

```

Comment:
this change is not needed in remote. for current release the publish command would need to be run locally - so you can simply apply the "packages" change locally to control which packages to publish